index.html

```diff
@@ -4,42 +4,210 @@
-          <li><i class='bx bxl-gmail'></i></li>
-          <li><i class='bx bxl-github'></i></li>
+    <section class="second-paper">
+        <div class="page1">
+        <div class="snapshoot">
+            <img src="./Snapshoot Portfolio.svg" alt="snap">
+        </div>
+        <div class="left-block">
+            <div class="primary-text1"></div>
+        <h1 class="project-title">Tonic</h1>
+        <ul class="frame1">
+            <div class="client"><li class="text1">canopy</li></div>
+            <div class="role"><li class="text2">Back End Dev</li></div>
+            <div class="year"><li class="text3">2015</li></div>
         </ul>
+        <div class="primary-text2">
+        <p> A daily selection of privately personalised reads; no accounts or sign up required </p>
+        </div>
+        <div class="tag">
+            <div class="tag1"><button class="tag-text1">html</button></div>
+            <div class="tag2">
+               <button class="tag-text2">css</button>
+            </div>            
+            <div class="tag3">
+               <button class="tag-text3">javascript</button>
+            </div>            
+
+        </div>
+        <div class="action">
+            <div class="normal-button">
+                <input class="action-text" type="text" value="see project">
+            </div>
         </div>
     </div>
+       </div>
+       <div class="page2">
+            <div class="snapshoot2">
+                <img src="./Snapshoot Portfolio2.svg" alt="snap">
+            </div>
+            <div class="left-block2">
+                <div class="primary-text21"></div>
+            <h1 class="project-title2">Multi-Post stories</h1>
+            <ul class="frame2">
+                <div class="client2"><li class="text21">canopy</li></div>
+                <div class="role2"><li class="text22">Back End Dev</li></div>
+                <div class="year2"><li class="text23">2015</li></div>
+            </ul>
+            <div class="primary-text22">
+            <p> A daily selection of privately personalised reads; no accounts or sign-up required. </p>
+            </div>
+            <div class="tags">
+                <div class="tags1"><button class="tag-texts1">html</button></div>
+                <div class="tags2">
+                   <button class="tag-texts2">css</button>
+                </div>            
+                <div class="tags3">
+                   <button class="tag-texts3">javascript</button>
+                </div>            
+    
+            </div>
+            <div class="action2">
+                <div class="normal-button2">
+                  <input class="action-text2" type="text" value="see project">
+                </div>
+            </div>
+        </div>
+           </div>
+           <div class="page3">
+            <div class="snapshoot3">
+            <img src="./Snapshoot Portfolio 3.svg" alt="snap">
+            <div class="left-block2">
```

Comment:
- [ ] Kindly fix the attribute for img tag `src` value not correctly located and why it affects the design. Please fix it :wrench: 

Example

```
  img src="./snapshoot/potfolio.svg"
```